src/components/timeTracker.tsx

```diff
@@ -1,10 +1,21 @@
-const TimeTracker = ({ timeStamp }) => {
-  const currentTime = new Date().getTime();
-  const refreshTime = currentTime - timeStamp;
-  const timeInMin = Math.round(refreshTime / 60000);
+import { useEffect, useState } from "react";
+
+interface TimeTrackerProps {
+  timeStamp: number;
+  classname?: string;
+}
+
+const TimeTracker = (props: TimeTrackerProps) => {
+  const [timeInMin, setTimeInMin] = useState<number>(0);
+
+  useEffect(() => {
+    const currentTime = new Date().getTime();
+    const refreshTime = currentTime - props.timeStamp;
+    setTimeInMin(Math.round(refreshTime / 60000));
+  }, [new Date().getTime()]);
 
   return (
-    <span className="text-xs text-[#8B8C8F] mr-3">
+    <span className={`${props.classname} text-xs text-[#8B8C8F] mr-3`}>
```

Comment:
`classname` is not being passed. so why are u keeping it ?

Reply:
classname is an optional prop added to make the component reusable with different styles.